### ISSUE TITLE
Would you like to (help) maintain this plugin?

### ISSUE THREAD

**Issue by webpro** [release-it/conventional-changelog#104]
I could use some help maintaining this plugin. The thing is, I don't really use it myself.

Feel free to respond or DM me about this, I'm sure can figure something out. Thanks a lot in advance!

**Comment by Farenheith** [comment]
Hi! I'll try to fix https://github.com/release-it/conventional-changelog/issues/128 at my fork, ok? I'd be happy to be a contributor as I'm active user of this plugin

**Comment by webpro** [comment]
@Farenheith Of course, please go ahead I'm happy to review PRs (and publish new releases).
